include/image-commands.mk

```diff
@@ -95,10 +95,7 @@ endef
 define Build/append-squashfs-fakeroot-be
 	rm -rf $@.fakefs $@.fakesquashfs
 	mkdir $@.fakefs
-	$(STAGING_DIR_HOST)/bin/mksquashfs-lzma \
```

Comment:
are you sure, squashfs-ng supports squashfs3 in this special flavour?

Reply:
No, I'll revert this